From f7f53bac2801cbddffcc36b0af0a5a8d7509b777 Mon Sep 17 00:00:00 2001
From: Clayton Craft <clayton@craftyguy.net>
Date: Fri, 1 Apr 2022 13:57:06 -0700
Subject: [PATCH] abuild.conf: disable Go's buildvcs

This was added in Go 1.18 and enabled by default, and causes builds to
fail if $builddir is not a repo.
---
 abuild.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.conf b/abuild.conf
index 4ff34f1..e0270ce 100644
--- a/abuild.conf
+++ b/abuild.conf
@@ -2,7 +2,7 @@ export CFLAGS="-Os -fomit-frame-pointer"
 export CXXFLAGS="$CFLAGS"
 export CPPFLAGS="$CFLAGS"
 export LDFLAGS="-Wl,--as-needed,-O1,--sort-common"
-export GOFLAGS="-buildmode=pie"
+export GOFLAGS="-buildmode=pie -buildvcs=false"
 # Do note that these should work with at least GDC and LDC
 export DFLAGS="-Os"
 
